extensions/amp-subscriptions-google/0.1/amp-subscriptions-google.js

```diff
@@ -465,18 +488,125 @@ export class GoogleSubscriptionsPlatform {
    * @return {boolean}
    */
   isPingbackEnabled() {
-    return false;
+    // Only enable pingback if useful deferred account creation.
+    return (
+      this.serviceConfig_['hasAssociatedAccountUrl'] &&
+      this.serviceConfig_['accountCreationRedirectUrl']
+    );
   }
 
   /** @override */
   pingbackReturnsAllEntitlements() {
-    return false;
+    return true;
   }
 
   /**
-   * Performs the pingback to the subscription platform
-   */
-  pingback() {}
+   * @param {?./entitlement.Entitlement} selectedEntitlement
+   * @return {!Promise<{found: boolean}>}
+   * @private
+   * */
+  hasAssociatedUserAccount_(selectedEntitlement) {
+    let storage;
+    return new Promise((resolve) => {
+      // First try to fetch from storage
+      this.storage_
+        .then((s) => {
+          storage = s;
+          return s.get(LOCAL_STORAGE_KEYS.HAS_PUBLISHER_ACCOUNT);
+        })
+        .then((publisherAccountLocalValue) => {
+          if (publisherAccountLocalValue !== undefined) {
+            // We found a cached value for the API call, return.
+            resolve({found: publisherAccountLocalValue});
+            return;
+          }
+          const hasAssociatedAccountUrl = /** @type {string} */ (devAssert(
+            this.serviceConfig_['hasAssociatedAccountUrl'],
+            'hasAssociatedAccountUrl is null'
+          ));
+
+          // Do the actual call and then store the value in storage
+          this.urlBuilder_
+            .buildUrl(hasAssociatedAccountUrl, /* useAuthData */ true)
+            .then((url) =>
+              this.xhr_.fetchJson(url, {
+                method: 'POST',
+                credentials: 'include',
+                body: {
+                  entitlements: this.serviceAdapter_.stringifyForPingback(
+                    selectedEntitlement
+                  ),
+                },
+              })
+            )
+            .then((result) => result.json())
+            .then((jsonResult) => {
+              console.log(jsonResult);
+              storage
+                .set(LOCAL_STORAGE_KEYS.HAS_PUBLISHER_ACCOUNT, jsonResult.found)
+                .then(() => {
+                  resolve(jsonResult);
+                });
+            });
+        });
+    });
+  }
+
+  /**
+   * @override
+   * @param {?./entitlement.Entitlement} entitlements
+   * @return {!Promise|undefined}*/
+  pingback(entitlements) {
+    if (!this.isPingbackEnabled() || !entitlements) {
+      return;
+    }
+    return new Promise((resolve, unusedReject) => {
+      // Check if user has denied account creation already
+      let storage;
+      this.storage_
+        .then((s) => {
+          storage = s;
+          return s.get(LOCAL_STORAGE_KEYS.HAS_REJECTED_ACCOUNT_CREATION);
```

Comment:
This is remembered forever? Is this really "Never ask me again"? Or more of a "not now"?

Reply:
I think probably a now now is a the best way to go - perhaps a default value (30 days?  a week?) and a publisher override in the config?